src/include/common/logger.h

```diff
@@ -26,8 +26,8 @@
 
 // Fix for PRId64 (See https://stackoverflow.com/a/18719205)
 #if defined(__cplusplus) && !defined(__STDC_FORMAT_MACROS)
-#define __STDC_FORMAT_MACROS 1 // Not sure where to put this
-#endif 
+#define __STDC_FORMAT_MACROS 1  // Not sure where to put this

```

Comment:
[trivial] Is this a auto-formatting change?

Reply:
Yes